kong/plugins/http-log/handler.lua

```diff
@@ -93,6 +93,10 @@ local function send_payload(self, conf, payload)
   params_cache.keepalive_timeout = keepalive
 
   local url = fmt("%s://%s:%d%s", parsed_url.scheme, parsed_url.host, parsed_url.port, parsed_url.path)
+  
+  if parsed_url.query then
+    url = url .. "? ".. parsed_url.query
```

Comment:
I think the space after the question mark was meant to be outside of the quotes:
```suggestion
    url = url .. "?" .. parsed_url.query
```